src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -728,15 +728,6 @@ rpmostree_compose_builtin_tree (int             argc,
 
       self->workdir = g_file_new_for_path (tmpd);
       workdir_is_tmp = TRUE;
-
-      if (opt_workdir_tmpfs)
```

Comment:
Should we at least print a deprecation notice of some sort? (As well as in the option help string)?